### PR TITLE
Merge upstream 2021-09-10

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

**Comment by honza** [comment]
`ci/prow/unit` is failing because the image is using golang-1.14 which is too old

**Review by sadasu — COMMENTED** [review]
Thanks for adding the downstream-only fix to disable webhooks.

File: .ci-operator.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: golang-1.14
+  tag: golang-1.16
```
Comment (hardys) [inline .ci-operator.yaml:4]: I think we should land https://github.com/openshift/baremetal-operator/pull/177/ instead then rebase, right?

**Pull request merged**